### PR TITLE
test: run test-nss-users/hosts by default

### PR DESCRIPTION
**Pull request by yuwata**
Prompted by https://github.com/systemd/systemd/issues/21935#issuecomment-1003214716.

### CODE REVIEW

**Comment by thesamesam** [comment]
Excellent news. Without #21944, the test suite fails on sparc at least (we're yet to see if the sanitizers in CI would pick it up though):
```
616/648 test-nss-hosts                            OK               5.28s
617/648 test-nss-users                            FAIL             5.24s   killed by signal 10 SIGBUS
>>> PATH=/var/tmp/portage/sys-apps/systemd-250-r1/work/systemd-250-.sparc64:/var/tmp/portage/sys-apps/systemd-250-r1/temp/python3.9/bin:/usr/lib/portage/python3.9/ebuild-helpers/xattr:/usr/lib/portage/p
ython3.9/ebuild-helpers:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/opt/bin MALLOC_PERTURB_=163 SYSTEMD_LANGUAGE_FALLBACK_MAP=/var/tmp/portage/sys-apps/systemd-250-r1/work/systemd-250/
src/locale/language-fallback-map /var/tmp/portage/sys-apps/systemd-250-r1/work/systemd-250-.sparc64/test-nss-users
```

File: src/test/meson.build [hunk 1]
```diff
@@ -608,14 +608,14 @@ tests += [
           'src/test/nss-test-util.h'],
          [],
          [libdl],
-         [], 'ENABLE_NSS', 'manual'],
+         [], 'ENABLE_NSS'],
```
Comment (evverx) [inline src/test/meson.build:611]: @yuwata I'd wait for @keszybz to weigh in. I'm still not sure why those tests were "manual".

Comment (evverx) [reply src/test/meson.build:611]: Looking at https://the-real-systemd.semaphoreci.com/jobs/f966d40a-b3c7-4414-a741-684d7df60e1b where the tests failed with
```
 FAIL: test-nss-hosts18:06
======== myhostname ========18:06
Failed to load module myhostname: libnss_myhostname.so.2: cannot open shared object file: No such file or directory
======== mymachines ========19:06
Failed to load module mymachines: libnss_mymachines.so.2: cannot open shared object file: No such file or directory
```
It seems they need polishing

Comment (Arfrever) [reply src/test/meson.build:611]: Probably test suite should set `LD_LIBRARY_PATH` variable pointing to directory with locally built `libnss_myhostname.so.2` and `libnss_mymachines.so.2`.

**Comment by evverx** [comment]
@yuwata I hopefully fixed those tests in https://github.com/systemd/systemd/pull/21778.

**Pull request closed without merging**